app/javascript/react/components/ProductShow.js

```diff
@@ -1,16 +1,31 @@
-import React from 'react'
+import React from "react";
+import Review from "./Review";
 
-const ProductShow = ({ id, name, description, url, image_url }) => {
-  return(
-    <div className="">
+const ProductShow = ({ id, name, description, url, image_url, reviews }) => {
+  const productReviews = reviews.map((review) => {
+    return (
+      <div key={review.id}>
+        <Review
+          id={review.id}
+          rating={review.rating}
+          body={review.body}
+          user={review.commenter_name}
+        />
+      </div>
```

Comment:
if you want this div for each review, it should be inside the Review component, or removed.